src/Host.h

```diff
@@ -575,9 +575,13 @@ class Host : public QObject
     bool mLogStatus;
     bool mEnableSpellCheck;
     QStringList mInstalledPackages;
+    // module name = location on disk, sync to other profiles?, priority
```

Comment:
```suggestion
    // Key: module name
    // Values:
    // 0: location (pathFileName) on disk,
    // 1: sync ("0" = No, Any other number as a string = Yes), to other profiles?,
    // 2: priority (Integer number as a string - may NOT be present)
```

Note that the third piece of information may never be there, it is confusing to determine, but from casual inspection of the code I can see, e.g.:
* at around line 964 in `(void) XMLimport::readHostPackage(Host*)` it is read from the external source but not stored there in the temporary `(QStringList) moduleList` that is added to this member
* at around 1736 in `(bool) Host::installPackage(const QString&, int)` that detail is not stored in the `QStringList` value and it isn't immediately clear where it is added (if at all) - as well as the detail seemingly being stored separately in `QMap<QString, int> mModulePriorities;` a couple of lines down...

:confused: